sellercenter-product-port/src/main/java/com/gfg/sellercenter/product/port/service/ProductReaderService.java

```diff
@@ -29,7 +30,8 @@
     @NotNull
     private final JsonHttpReader jsonReader;
 
-    private static final String API_PATH = "/api/products/v1";
+    private static final String API_PATH_PRODUCTS_BY_IDS = "/api/products/v1";
```

Comment:
here we can also include this `?product_ids=`. because the name of the constant says that it's only for looking up products by Ids

Reply:
I did like you suggested. Although I would appreciate comment from @mmirboboev about it. I guess idea about those constants is to keep path only and add parameters in body of methods.